### DISCUSSION TITLE
filename completion

### DISCUSSION THREAD

**Discussion by BelT327l** [chrisant996/clink#633] [General]
I use clink into cmder on different pcs and the problem is the auto-completion on filenames... it shoud be like filling the name of the file completely AND a point, but NOT filling the first alphabetical filetype... For example many multiple files with the same name but different ending: 1234abc.AAC 1234abc.mkv 1234abc.WEBM... if i write 1234 if there are no similar filenames after that, it should automatically stop at 1234abc. and thats it... not 1234abc.AAC ...
On the other pc that behaviour is as like i wanted it but here it is not... i do not know what i made differently (i did nothing, it was the installer of cmder)...

**Comment by chrisant996** [comment]
It sounds like you want the key binding for <kbd>Tab</kbd> to be [`complete`](https://chrisant996.github.io/clink/clink.html#rlcmd-complete), but on one machine it's bound to something else like maybe [`old-menu-complete`](https://chrisant996.github.io/clink/clink.html#rlcmd-old-menu-complete).

Cmder doesn't mess with key bindings.

Here is a good place in the Clink documentation to start, for figuring out why it's behaving the way it is and how to change it:
- https://chrisant996.github.io/clink/clink.html#completion-keys

Here is documentation about key bindings in general:
- https://chrisant996.github.io/clink/clink.html#keybindings
- https://chrisant996.github.io/clink/clink.html#init-file
- https://chrisant996.github.io/clink/clink.html#readline-key-bindings
